### DISCUSSION TITLE
Lesson 7 - s_priceFeed : could we set it to immutable ?

### DISCUSSION THREAD

**Discussion by getzelus** [Cyfrin/foundry-full-course-cu#638] [General]
At 37:00 of the second video (lesson 7), the **FundMe.sol** contract has a variable : 
`AggregatorV3Interface private s_priceFeed `
Can we set it to _immutable_ :
`AggregatorV3Interface private immutable i_priceFeed `

 cause it only changes in the constructor :
```
 constructor(address priceFeed) {
        i_owner = msg.sender;
        s_priceFeed = AggregatorV3Interface(priceFeed);
    }
```

So it would be better for gaz optimization ?

**Comment by theirrationalone** [comment]
Voilaaaa, why not, nowhere it was reassigned after that first assignment in the `constructor`
